index.js

```diff
@@ -0,0 +1,12 @@
+const {randomStudentsFactory, createRandomStudent} = require("./student");
+const { writeJSONFile, readJSONFile } = require("./helper");
+
+function run() {
+    if(process.argv[3]) {
+    console.log(randomStudentsFactory(process.argv[3]));
```

Comment:
Be careful here - you are console.log'n the execution of a function taht will write data.  It may _seem_ like this should not change you data because it is in console log but remember writeFileSync() will change that file regardless if it is in the console.log()